package.json

```diff
@@ -53,19 +51,18 @@
     "@rollup/plugin-node-resolve": "^15.2.3",
     "@rollup/plugin-replace": "^5.0.5",
     "@rollup/plugin-typescript": "^11.1.6",
-    "@types/diff": "^5.0.9",
     "@types/mime-types": "^2.1.4",
     "@types/node": "^18.17.15",
     "@types/semver": "^7.5.6",
-    "@types/vscode-semver": "npm:@types/semver@=5.5.0",
     "@types/vscode": "~1.84.2",
```

Comment:
Why not `1.85.0`?

Reply:
The types weren't available at the time the update was make, and it's excluded for ncu so I didn't noticed it, will update

Reply:
5ada4c15afcb1d475adc51a202647290c0a4ce81